src/main/java/services/update_entities/UpdateEvent.java

```diff
@@ -0,0 +1,32 @@
+package services.update_entities;
+
+import data_gateway.CalendarManager;
+import data_gateway.EventEntityManager;
+
+import java.time.LocalTime;
+
+public class UpdateEvent {
```

Comment:
It seems that you are creating an `UpdateEvent` class whenever someone wants to update an entity. But it makes more sense to me that this class should have methods that take in `id` as input and make the updates from there, so there will be only one existence of this class in the whole program, like our other use case classes.